runtime/interpreter/dynamictype.go

```diff
@@ -40,24 +41,40 @@ type MetaTypeDynamicType struct{}
 
 func (MetaTypeDynamicType) IsDynamicType() {}
 
+func (MetaTypeDynamicType) IsImportable() bool {
+	return false
```

Comment:
Should be true, maybe just refer to use the same value as in `sema`, here and in the other implementations, i.e.:

```suggestion
	return sema.MetaType.Importable
```